ambry-clustermap/src/test/java/com.github.ambry.clustermap/HelixClusterManagerTest.java

```diff
@@ -310,8 +310,7 @@ public void emptyPartitionOverrideTest() throws Exception {
 
     Set<String> writableInClusterManager = new HashSet<>();
     for (PartitionId partition : clusterManagerWithEmptyRecord.getWritablePartitionIds(null)) {
-      String partitionStr =
-          useComposite ? ((Partition) partition).toPathString() : ((AmbryPartition) partition).toPathString();
+      String partitionStr = useComposite ? partition.toPathString() : partition.toPathString();
```

Comment:
minor: looks like it can be simplified as `String partitionStr = partition.toPathString()`

Reply:
fixed.